include/ibamr/IBFEMethod.h

```diff
@@ -762,6 +844,32 @@ class IBFEMethod : public IBStrategy
      */
     void initializeVelocity(unsigned int part);
 
+    /*!
+     * Get the forward transfer schedule (i.e., from the main hierarchy to the
+     * scratch hierarchy) associated with the given level and index. If
+     * necessary the schedule is created and stored in a map.
+     *
+     * If needed, a SAMRAI::xfer::RefinePatchStrategy object can be provided
+     * for filling ghost data at physical boundaries.
+     */
+    SAMRAI::xfer::RefineSchedule<NDIM>&
+    getForwardScratchSchedule(const int level_number,
+                              const int data_idx,
+                              SAMRAI::xfer::RefinePatchStrategy<NDIM>* rps = nullptr);
```

Comment:
```suggestion
                              SAMRAI::xfer::RefinePatchStrategy<NDIM>* patch_strategy = nullptr);
```
or `refine_patch_strategy`

Reply:
Instead of "forward" and "backward", can we try to make these function names include something like "to scratch hierarchy" and "from scratch hierarchy"?

Another alternative would be to have a single function that has a mode that could be either `FORWARD` or `REVERSE` (similar to PETSc data transfer functions).

Reply:
I went with `getPrimaryToScratchSchedule` and `getScratchToPrimarySchedule`.